doxia-site-renderer/pom.xml

```diff
@@ -161,13 +165,13 @@ under the License.
     <dependency>
       <groupId>net.sourceforge.htmlunit</groupId>
       <artifactId>htmlunit</artifactId>
-      <version>2.7</version>
+      <version>2.24</version>
```

Comment:
This needs a separate ticket.

Reply:
Not practical alas, as explained in the commit the change in htmlunit required a change to the xhtml5 patch to account for changes in htmlunit behaviour.